src/mower_logic/src/mower_logic/mower_logic.cpp

```diff
@@ -434,15 +434,15 @@ void checkSafety(const ros::TimerEvent &timer_event) {
             // set this if we don't even have an orientation
             high_level_status.gps_quality_percent = -1;
         }
-        ROS_WARN_STREAM_THROTTLE(1,"Low quality GPS");
+        ROS_WARN_STREAM_THROTTLE(last_config.gps_message_throttle,"Low quality GPS");
```

Comment:
Could leave it as it but  just not log it when `!currentBehavior->needs_gps())`?